.github/workflows/close-when-stale.yaml

```diff
@@ -0,0 +1,34 @@
+name: Check for stale issues
+on:
+  schedule:
+    - cron: '37 21 * * *' # at 21:37 every day
+  issues:
+    types: [edited]
+  issue_comment:
+    types: [created, edited]
+  workflow_dispatch:
+
+jobs:
+   main:
+    if: github.repository == 'software-mansion/radon-ide'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        uses: actions/checkout@v4
+        with:
+          repository: 'software-mansion-labs/swmansion-bot'
+          ref: stable
+
+      - uses: actions/cache@v4
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install Actions
+        run: yarn install
+
+      - name: Close when stale
+        uses: ./close-when-stale
+        with:
+          close-when-stale-label: 'Close when stale'
```

Comment:
We don't have this label in our project though. Also all labels that we have starts with lowercase letters, so perhaps we want to create one that aligns with the existing labels in that matter

Reply:
True, I created lowercase label and aligned the code in: https://github.com/software-mansion/radon-ide/pull/807/commits/f077e87fa903fa1f8891cc007a6c3397d9df8330